app/assets/javascripts/admin/controllers/admin-customize-themes-show.js.es6

```diff
@@ -60,6 +78,21 @@ export default Controller.extend({
     }
   },
 
+  @discourseComputed("model.parentThemes.[]")
+  parentThemesNames(parentThemes) {
+    return parentThemes.mapBy("name");
+  },
```

Comment:
why not use the `mapBy` computed directly ? 

http://api.emberjs.com/ember/3.13/functions/@ember%2Fobject%2Fcomputed/mapBy